### ISSUE TITLE
Pull requests?

### ISSUE THREAD

**Issue by extesy** [superlime/hoverzoomgold#1]
Thanks for forking my project. How about sending some pull requests upstream?

Also I'd like you to explicitly mention differentiation between your fork and upstream project in extension description. I've already received issue reports from users of your fork and I don't want to spend my time on those.

**Comment by superlime** [comment]
Ah!  Sorry about that!  The first time I submitted the extension I went in to great detail about how it was forked from HoverFree, and just pulled in the latest updates from HoverZoom and added the reddit gold functionality...but apparently describing in detail why mine was different triggered it getting blocked for being just a resubmission of the existing extension.  It didn't get accepted until I removed most of those details from the description and just added them to the github readme.  Sometime after that I'd moved from HoverFree to your fork since you were updating more frequently, but apparently I didn't update the readme afterwards.

I hadn't made a pull request of my changes because mine is a pretty specialized use case (it's really just adding reddit gold functionality), and it's kinda awkwardly hacked in in the main loading function.  Figured it was too messy for you to be interested in taking.
